sessionctx/variable/session.go

```diff
@@ -97,40 +97,41 @@ func (tc *TransactionContext) UpdateDeltaForTable(tableID int64, delta int64, co
 
 // SessionVars is to handle user-defined or global variables in current session.
 type SessionVars struct {
-	// user-defined variables
+	// Users are user defined variables.
 	Users map[string]string
-	// system variables
+	// Systems are system variables.
 	Systems map[string]string
-	// prepared statement
+	// PreparedStmts stores prepared statement.
 	PreparedStmts        map[uint32]interface{}
 	PreparedStmtNameToID map[string]uint32
-	// prepared statement auto increment id
+	// preparedStmtID is id of prepared statement.
 	preparedStmtID uint32
 
 	// retry information
 	RetryInfo *RetryInfo
 	// Should be reset on transaction finished.
```

Comment:
what does this sentence mean? when transaction finished? or "transaction finished" refers to "finished/completed transaction"?